.editorconfig

```diff
@@ -0,0 +1,23 @@
+
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+
+# Drupal Coding Standards
+# https://www.drupal.org/docs/develop/standards
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 2
```

Comment:
@coreylafferty These two lines are different from the WP version because Drupal coding standards use two spaces for indent: https://www.drupal.org/docs/develop/standards/coding-standards#indenting